### ISSUE TITLE
Adding the ability to remember the last selected profile when restarting vscode instead of using the default profile.

### ISSUE THREAD

**Issue by Natizyskunk** [Natizyskunk/vscode-sftp#393]
It could be an optional parameter like `"rememberProfile": true`.

**Comment by satiromarra** [comment]
@Natizyskunk  I think it would be better to use "workspaceState" for this, so the file "sftp.json" would not have to be modified

**Comment by Natizyskunk** [comment]
@satiromarra That's actually sound great. Do you have any idea on how we could implement to make it easy as possible to maintain ?

**Comment by satiromarra** [comment]
@Natizyskunk Yes, I have an idea of what it should be like. Deleting the operation in fileService when it is chosen and when it is instantiated.
